test/transport.browser.spec.ts

```diff
@@ -16,10 +19,51 @@ function ignoredDialOption (): CreateListenerOptions {
 
 describe('WebRTC Transport', () => {
   let components: underTest.WebRTCTransportComponents
+  let metrics: Metrics
 
   before(async () => {
+    metrics = stubObject<Metrics>({
```

Comment:
I wanted to call out that this piece is arguably not super useful with the tests in their current state.  I added this here for completeness' sake. If we wish to add additional metrics or add full e2e testing, we can test that the metrics are invoked further.